samples/tab-conversations/csharp/TabConversation/note.txt

```diff
@@ -0,0 +1,7 @@
+Remove below images for Images folder 
```

Comment:
Read this text file and act on it.

Don't forget to remove it.

Reply:
Done, removed them